robottelo/ssh.py

```diff
@@ -50,18 +50,7 @@ def command(
         password=password,
         port=port,
     )
-    try:
-        result = client.execute(cmd, timeout=timeout)
-    except (exceptions.Timeout, exceptions.SocketSendError):
-        # If the client is in a bad state, invalidate the cache and get a new one
-        get_client.cache_clear()
-        client = get_client(
-            hostname=hostname,
-            username=username,
-            password=password,
-            port=port,
-        )
-        result = client.execute(cmd, timeout=timeout)
+    result = client.execute(cmd, timeout=timeout)
```

Comment:
Approved, but gotta remove the unused exceptions import

Reply:
ahhh true!